src/components/Chatbar/Chatbar.tsx

```diff
@@ -96,12 +94,13 @@ export const Chatbar = () => {
       isOpen={showChatbar}
       itemComponent={<Conversations conversations={filteredConversations} />}
       folderComponent={<ChatFolders />}
-      folders={folders}
-      items={conversations}
       filteredItems={filteredConversations}
       searchTerm={searchTerm}
-      handleSearchTerm={(searchTerm: string) =>
-        dispatch(ConversationsActions.setSearchTerm({ searchTerm }))
+      searchFilters={searchFilters}
+      handleSearchTerm={(searchTerm: string, searchFilters: SearchFilters) =>
```

Comment:
What do you think about extracting this into the separate function for better readability?